app/client/components/pages/mapPage.jsx

```diff
@@ -1,97 +1,41 @@
-import React, { Suspense, lazy } from "react";
+import React, { lazy } from "react";
 import { connect } from "react-redux";
-import {
-  CenterLoadingSideBarActive,
-  WrapperDiv,
-  PageWrapper
-} from "../styledElements";
+import { WrapperDiv, PageWrapper } from "../styledElements";
 import LoadingSpinner from "../loadingSpinner.jsx";
-import { LoadCreditTransferAction } from "../../reducers/creditTransfer/actions";
-import { AllowedFiltersAction } from "../../reducers/allowedFilters/actions";
-import NoDataMessage from "../NoDataMessage";
+import FilterModule from "../filterModule/FilterModule";
 
 const HeatMap = lazy(() => import("../heatmap/heatmap.jsx"));
 
 const mapStateToProps = state => {
   return {
-    creditTransfers: state.creditTransfers,
-    login: state.login
-  };
-};
-
-const mapDispatchToProps = dispatch => {
-  return {
-    loadCreditTransferList: query =>
-      dispatch(
-        LoadCreditTransferAction.loadCreditTransferListRequest({ query })
-      ),
-    loadAllowedFilters: () =>
-      dispatch(AllowedFiltersAction.loadAllowedFiltersRequest())
+    metricsLoadStatus: state.metrics.loadStatus
   };
 };
 
 class MapPage extends React.Component {
   constructor() {
     super();
-    this.state = {};
-  }
-
-  componentDidMount() {
-    let transfer_type = "ALL";
-    let per_page = 50;
-    let page = 1;
-    this.props.loadCreditTransferList({
-      get_stats: true,
-      transfer_type: transfer_type,
-      per_page: per_page,
-      page: page
-    });
-
-    this.props.loadAllowedFilters();
   }
 
   render() {
-    if (this.props.creditTransfers.loadStatus.isRequesting === true) {
-      return (
-        <WrapperDiv>
-          <CenterLoadingSideBarActive>
-            <LoadingSpinner />
-          </CenterLoadingSideBarActive>
-        </WrapperDiv>
-      );
-    } else if (Object.values(this.props.creditTransfers.byId).length === 0) {
-      return (
+    return (
+      <WrapperDiv>
         <PageWrapper>
-          <NoDataMessage />
+          <FilterModule
+            defaultGroupBy={"sender,coordinates"}
+            filterObject={"credit_transfer"}
+            dateRange={null}
+            hideGroupBy={true}
+          />
```

Comment:
Small change to improve styling
![Screen Shot 2020-12-15 at 5 48 14 pm](https://user-images.githubusercontent.com/27039316/102180907-bc326780-3efd-11eb-90c3-ab6359ce1af8.png)